front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ComponentType, useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { updateSpeedLimitByTag } from 'reducers/osrdconf';
+import { setFailure } from 'reducers/main';
+import icon from 'assets/pictures/components/speedometer.svg';
+import SelectImprovedSNCF from 'common/BootstrapSNCF/SelectImprovedSNCF';
+import { getInfraID, getSpeedLimitByTag } from 'reducers/osrdconf/selectors';
+import { Dispatch } from 'redux';
+import { noop } from 'lodash';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/query/react';
+import './SpeedLimitByTagSelector.scss';
+
+type SpeedLimitByTagSelectorProps = {
+  condensed?: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  speedLimitByTag?: string;
+  speedLimitsByTagsFromApi?: string[];
+  dispatchUpdateSpeedLimitByTag?: (tag: string) => Dispatch | void;
+};
```

Comment:
```suggestion
  speedLimitByTagText: string;
};
```
To avoid crashing the navigator, we can pass the text to display as an argument to avoid calling useTranslate in the isolatedComponent (not the best solution, but I can't see how we can do it cleaner right now)

Reply:
chose another solution